adapters/bmtm/params_test.go

```diff
@@ -2,33 +2,35 @@ package bmtm
 
 import (
 	"encoding/json"
+	"fmt"
 	"testing"
 
 	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestValidParams(t *testing.T) {
 	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
 	if err != nil {
-		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+		assert.NoError(t, err, fmt.Sprintf("Failed to fetch the json-schemas: %s", err.Error()))
 	}
 
 	for _, validParam := range validParams {
 		if err := validator.Validate(openrtb_ext.BidderBmtm, json.RawMessage(validParam)); err != nil {
-			t.Errorf("Schema rejected brightMountainMedia params: %s", validParam)
+			assert.NoError(t, err, fmt.Sprintf("Schema rejected brightMountainMedia params: %s", validParam))
 		}
 	}
 }
 
 func TestInvalidParams(t *testing.T) {
 	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
 	if err != nil {
-		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+		assert.NoError(t, err, fmt.Sprintf("Failed to fetch the json-schemas: %s", err.Error()))
 	}
 
 	for _, invalidParam := range invalidParams {
 		if err := validator.Validate(openrtb_ext.BidderBmtm, json.RawMessage(invalidParam)); err == nil {
```

Comment:
You can remove the `err == nil` check here but you'll have to change the assert function below to `assert.Error` because in case of invalid params you do expect an error to occur